src/stories/SearchForm.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import {ComponentMeta, ComponentStory} from "@storybook/react";
+
+import {useState} from "react";
+
+import SearchForm from "../components/SearchForm";
+
+export default {
+    title: "Components/SearchForm",
+    component: SearchForm,
+} as ComponentMeta<typeof SearchForm>
+
+const Template: ComponentStory<typeof SearchForm> = args => {
+    const INITIAL_VALUE = "This is initial value";
```

Comment:
Such constant values can be placed outside the component as we don't need to inialize them at every render